api-reference/10 UI Components/dxValidationSummary/3 Methods/refreshValidationGroup().md

```diff
@@ -3,7 +3,252 @@ id: dxValidationSummary.refreshValidationGroup()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Refreshes the ValidationSummary subscription to a target ValidationGroup.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Use this method if target ValidationGroup is recreated.
+
+---
+##### jQuery
+    <!-- tab: index.html -->
+    <div id="container"></div>
+    <div id="summary"></div>
+    <div id="validate-group"></div>
+    <div id="remove-group"></div>
+    <div id="render-group"></div>
+
+    <!-- tab: index.js -->
+    $(function() {
+
+        function renderGroup() {
+            $("#container").append(
+                $("<div></div>").dxTextBox({}) 
+                .dxValidator({
```

Comment:
I'm not sure that using 2 same TextBoxes is a good idea. Theoretical utility of the sample is not changed but its simplicity and readability are worsened.

Reply:
We discussed it and decided to use one TextBox